cmd/kubeadm/app/util/config/testdata/validation/invalid_duplicate_field_nodecfg.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: NodeConfiguration
```

Comment:
`NodeConfiguration` does not exist in v1beta1. This needs to be `JoinConfiguration` and the file should be named `invalid_duplicate_field_joincfg.yaml`.
This is likely passing, because `JoinConfigFileAndDefaultsToInternalConfig` will return an error if there is no `JoinConfiguration` and you simply expect this to fail.